Framework/PythonInterface/plugins/algorithms/SaveP2D.py

```diff
@@ -0,0 +1,202 @@
+# Mantid Repository : https://github.com/mantidproject/mantid
+#
+# Copyright &copy; 2020 ISIS Rutherford Appleton Laboratory UKRI,
+#   NScD Oak Ridge National Laboratory, European Spallation Source,
+#   Institut Laue - Langevin & CSNS, Institute of High Energy Physics, CAS
+# SPDX - License - Identifier: GPL - 3.0 +
+
+import numpy as np
+import math
+from scipy.special import lambertw
+from mantid import mtd
+from mantid.api import (AlgorithmFactory, PythonAlgorithm, FileAction, FileProperty, WorkspaceProperty, Progress)
+from mantid.kernel import Direction
+
+
+class SaveP2D(PythonAlgorithm):
+    def category(self):
+        return 'Diffraction//DataHandling'
```

Comment:
I think these need to be backslashes 
```suggestion
        return 'Diffraction\\DataHandling'
```